.github/workflows/release.yml

```diff
@@ -29,6 +29,7 @@ jobs:
           tag: v0.0.19
           fileName: saturn-webui.tar.gz
           out-file-path: resources/webui
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
we were running into rate limiting issues because it was using a shared gha token instead of the repo one.